internal/ssm/source.go

```
@@ -78,6 +78,9 @@ func WithPublicKey(key string) SSMInstallerOption {
 // SSMInstaller provides a Source that retrieves the SSM installer from the official
 // release endpoint.
 func NewSSMInstaller(logger *zap.Logger, region string, opts ...SSMInstallerOption) Source {
+	if region == "" {
```

now that we have a default in the cmd level, can we remove this? just make region by mandatory
if you want, return error/panic if region is empty

this would reduce the chances of us breaking the flow and endup passing an empty string here and silently using the default region

---

will add to future pr